src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+@import '../utils/variables';
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image-container {
```

Якщо маєш вкладені селектори, то використовуй `&`
```suggestion
 &__image-container {
```